keyboards/handwired/yamsek/keymaps/default/keymap.c

```diff
@@ -0,0 +1,159 @@
+       ╚═══════════╩═══════════╩═══════════╣     .     ╠═══════════╬═══════════╣
+                                           ╚═══════════╣     /     ║           ║  Bot Row
+                                                       ╚═══════════╩═══════════╝
+                           ╔═════════╗
+                 ╔═════════╣3        ║  ╔═════════╗
+       ╔═════════╣2        ╠═════════╝  ║4        ║  Thumb Cluster key-order
+       ║1        ╠═════════╝  ╔═════════╬═════════╣
+       ╚═════════╝            ║6        ║5        ║
+                              ╚═════════╩═════════╝
+
+       +-----------+-----------+-----------+-----------+-----------+-----------+
+       |     Y     |     U     |     I     |     O     |     P     |     [     |  Top Row
+       +-----------+-----------+-----------+-----------+-----------+-----------+
+       |     H     |     J     |     K     |     L     |     ;     |     '     |  Mid Row
+       +-----------+-----------+-----------+-----------+-----------+-----------+
+       |     N     |     M     |     ,     |     .     |     /     |           |  Bot Row
+       +-----------+-----------+-----------+-----------+-----------+-----------+
+
+       +-----------+-----------+-----------+-----------+-----------+-----------+
+       +  Layer 1  |  Backsp   |   Delete  |  Space    |   Enter   |   Tab     | Thumb Cluster
+       +-----------+-----------+-----------+-----------+-----------+-----------+
+
+                   |           |           |           |           |
+                   |           |           |           |           |
+
+       LEFT
+                    ╔═════════╗
+       ╔═════════╗  ║4        ╠═════════╗
+       ║3        ║  ╚═════════╣5        ╠═════════╗
+       ╠═════════╬═════════╗  ╚═════════╣6        ║  Thumb Cluster key-order
+       ║2        ║1        ║            ╚═════════╝
+       ╚═════════╩═════════╝
+
+       +-----------+-----------+-----------+-----------+-----------+-----------+
+       |  Layer 4  |     Q     |     W     |     E     |     R     |     T     |  Top Row
+       +-----------+-----------+-----------+-----------+-----------+-----------+
+       |  Layer 3  |     A     |     S     |     D     |     F     |     G     |  Mid Row
+       +-----------+-----------+-----------+-----------+-----------+-----------+
+       |  Layer 2  |     Z     |     S     |     C     |     V     |     B     |  Bot Row
+       +-----------+-----------+-----------+-----------+-----------+-----------+
+
+       +-----------+-----------+-----------+-----------+-----------+-----------+
+       +  Layer 1  |  Backsp   |   Delete  |  L Shift  |   L Ctrl  |   L Alt   | Thumb Cluster
+       +-----------+-----------+-----------+-----------+-----------+-----------+
+
+                   |           |           |           |           |
+                   |           |           |           |           |*/
+    [BASE] = LAYOUT( // Base
```

Comment:
This needs to be updated.

Reply:
oh! thanks for spotting, updated with [e7cd2d2](https://github.com/qmk/qmk_firmware/pull/17902/commits/e7cd2d2ec5d4dbbba5a6ca5e0f6e134ac09a8ac3)